app/views/records/edit_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.input :rights, as: :select_with_modal_help,
```

Comment:
You are removing the help ? from the screen by removing that class.
![screen shot 2016-02-22 at 11 07 11 am](https://cloud.githubusercontent.com/assets/1599081/13223963/722d4e7a-d954-11e5-8424-8f6d131d7040.png)